srcpkgs/ydotool/files/ydotool.1

```diff
@@ -0,0 +1,285 @@
+.\" Generated by scdoc  1.11.2
```

Comment:
can we include your patch and generate this in the template instead?

Reply:
Thanks for reviewing :) I've got it using the patch and added my email to my account.

Reply:
I see you made a PR upstream, could you just import that patch including the headers? `wget https://github.com/ReimuNotMoe/ydotool/pull/168.patch -O srcpkgs/ydotool/man-update-for-1.0.1.patch`. I would remove the .diff and replace it with .patch, 99% of patches use .patch.